x-pack/plugins/apm/server/lib/rum_client/get_client_metrics.ts

```diff
@@ -21,18 +22,19 @@ export async function getClientMetrics({
 }: {
   setup: Setup & SetupTimeRange & SetupUIFilters;
 }) {
-  const projection = getRumOverviewProjection({
+  const projection = getRumPageLoadTransactionsProjection({
     setup,
   });
 
   const params = mergeProjection(projection, {
     body: {
       size: 0,
-      query: {
-        bool: projection.body.query.bool,
-      },
       aggs: {
-        pageViews: { value_count: { field: 'transaction.type' } },
+        pageViews: {
+          value_count: {
+            field: TRANSACTION_DURATION,
+          },
+        },
```

Comment:
any reason for this?

Reply:
It'll help when moving to aggregated transaction metrics. In that case, we'll use `transaction.duration.histogram`. I could also revert it, or entirely remove it. Is there a reason why you're using a `value_count` aggregation there? `transaction.type` will always be set, and will always be page-load in this situation, or is that interpretation wrong?

Reply:
no reason, i just needed a count of all filtered transactions

Reply:
Do you have any better agg in mind to get a count of all page-load transactions?

Reply:
In this case you could just use `track_total_hits`: true, and then use `response.hits.total.value` (AFAICT).

Reply:
Cool, yeah that can be done. We can do in separate PR. Don't want to hold this off so LGTM !!